cms/datasets/tests/test_blocks.py

```diff
@@ -0,0 +1,46 @@
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+from wagtail.blocks import StreamValue
+
+from cms.datasets.blocks import DatasetStoryBlock
+from cms.datasets.models import Dataset
+
+
+class TestDatasetStoryBlock(TestCase):
+    def setUp(self):
+        self.dataset = Dataset(
+            namespace="1",
+            edition="test1_edition",
+            version="test_version",
+            title="test_title",
+            description="test_description",
+            url="https://example.com",
+        )
+        self.dataset.save()
```

Comment:
```suggestion
        self.dataset = Dataset.objects.create(
            namespace="1",
            edition="test1_edition",
            version="test_version",
            title="test_title",
            description="test_description",
            url="https://example.com",
        )
```

Reply:
Addressed in https://github.com/ONSdigital/dis-wagtail/pull/123/commits/6ce4cf0620a7652c93c212421964b07cc5b5d86f